docker-compose.yml

```diff
@@ -25,12 +19,14 @@ services:
     ###
     #command: ["/wait/wait-for", "web:3000", "--", "nginx", "-g", "daemon off;"]
   web:
-    build: .
+    image: node:6.11.1
```

Comment:
Why don't we need to build from the dockerfile anymore?

Reply:
Because we have a common docker file for production and local environment! There are certain things that are not required for development builds which are present in the docker file. It will help cut down on production specific tasks and will help build development environment faster!